src/main/flight/dyn_notch_filter.c

```diff
@@ -387,10 +389,14 @@ static FAST_CODE_NOINLINE void dynNotchProcess(void)
         }
         case STEP_UPDATE_FILTERS: // 5.4us (2-9us) @ F722
         {
+            const float dtCompensation = schedulerGetCycleTimeMultiplier();
+            const float correctedLooptime = dynNotch.looptimeUs * dtCompensation;
+
             for (int p = 0; p < dynNotch.count; p++) {
                 // Only update notch filter coefficients if the corresponding peak got its center frequency updated in the previous step
+                // Correct the peak frequency based on the true dt
                 if (peaks[p].bin != 0 && peaks[p].value > sdftNoiseThreshold) {
-                    biquadFilterUpdate(&dynNotch.notch[state.axis][p], dynNotch.centerFreq[state.axis][p], dynNotch.looptimeUs, dynNotch.q, FILTER_NOTCH, 1.0f);
+                    biquadFilterUpdate(&dynNotch.notch[state.axis][p], dynNotch.centerFreq[state.axis][p] / dtCompensation, correctedLooptime, dynNotch.q, FILTER_NOTCH, 1.0f);
```

Comment:
This line corrects both the looptime for the notch filter and the peak inaccuracy due to looptime drifts.

Reply:
in biquad: `const float omega = 2.0f * M_PIf * filterFreq * refreshRate * 0.000001f;`

This only adds extra division and multiplication, without changing the result

Reply:
Oh, your right, I did a little more digging and yeah, the looptime inaccuracies and the notch update inaccuracies (due to looptime, do indeed cancel out). I'll be removing this from the PR.

Reply:
Pulled out all lowpass filters as their seems to be no appetite for it.